README.md

```diff
@@ -443,11 +443,11 @@ Some other useful tasks are:
 - `./gradlew fullAcceptanceTest`: run the ATs in the context of Docker
 - `./gradlew dbBootstrap`: set up the database
 - `./gradlew cleanDb`: clear down the database
-- `./gradlew startDockerContainer`: start the docker containers (and download images if needed)
-- `./gradlew stopAndRemoveDockerContainer`: shutdown the docker processes
+- `./gradlew startDockerContainer`: start the Docker containers (and download images if needed)
+- `./gradlew stopAndRemoveDockerContainer`: shutdown the Docker processes
 - `./gradlew startStoragesInDocker`: start the storage container (handy for running Nakadi directly or in your IDE)
 
-For working with an IDE, the `eclipse` IDE task is available and you'll be able to import the `build.gradle` into Intellij IDEA directly.
+For working with an IDE, the `eclipse` IDE task is available. You can directly import the `build.gradle` into IntelliJ.
```

Comment:
The "For working with an IDE" is the introduction for both the Eclipse and the IntelliJ parts. I'm not sure the new version  still works for this.